core/src/err/mod.rs

```diff
@@ -1099,7 +1099,7 @@ pub enum Error {
 	InvalidStorageVersion,
 
 	/// There was an outdated storage version stored in the database
-	#[error("The data stored on disk is out-of-date with this version. Please follow the upgrade guides in the documentation")]
+	#[error("The data stored on disk is out-of-date with this version. Please take a backup, and use the `surreal fix` command to upgrade the storage version.")]
```

Comment:
```suggestion
	#[error("The data stored on disk is out-of-date with this version. Please, back up your data and use the `surreal fix` command to upgrade the storage version.")]
```

Reply:
I think this may be a bit more correct-sounding, but I am not a native speaker either 😅

Reply:
```suggestion
	#[error("The data stored on disk is out-of-date with this version. Please back up your data and use the `surreal fix` command to upgrade the storage version.")]
```